ministryofjustice/correspondence_tool_staff
# CT-3035 Suppress case closure outcomes page

*Suppress case closure outcomes page for offender SARs.

## Description

For this PR, it is to remove the step of filling in information for closure outcome page when you close a case as it is irrelevant to offender-sar case type

## Self-review checklist


### Screenshots

before 
<img width="793" alt="Screenshot 2020-09-09 at 13 28 54" src="https://user-images.githubusercontent.com/26111671/92598671-0540a300-f2a1-11ea-857d-e6a36f5cc4cc.png">

after 
<img width="776" alt="Screenshot 2020-09-09 at 13 18 48" src="https://user-images.githubusercontent.com/26111671/92598333-8d727880-f2a0-11ea-82da-cb80f1a0ac24.png">


### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3035
### Deployment


### Manual testing instructions

Testing as Branson user, 
1. Login in as Branston user
2. Choose a open-late case (if not open late, you can change the date of receiving the case to a earlier date) and go through the whole flow until to the next step of closing the case
3. When you can click 'close' and choose today is as the closed date, 
4 After you confirm by clicking 'continue', you should go back to the case detail straightway with a message indicating the closure of this case

Testing as London user,  the flow shouldn't be affected
4. sign out
5. Login as London user
6. Repeat 2, 3
7. After you confirm by clicking 'continue', you should see a extra step for asking you to fill in some information for closure outcome
